cdap-watchdog/src/main/java/co/cask/cdap/logging/read/AvroFileReader.java

```diff
@@ -181,9 +174,34 @@ public void readLog(Location file, Filter logFilter, long fromTimeMs, long toTim
     }
   }
 
+  private List<LogEvent> readToNextSync(Filter logFilter, long fromTimeMs, DataFileReader<GenericRecord> dataFileReader,
```

Comment:
As we read from current sync to nextSync (current single block), I think it would be appropriate to use readToNext() (meaning read till end of current block)?